topdown/walk.go

```diff
@@ -70,6 +80,31 @@ func walk(filter, path *ast.Array, input *ast.Term, iter func(*ast.Term) error)
 	return nil
 }
 
+func walkNoPath(path, input *ast.Term, iter func(*ast.Term) error) error {
+	if err := iter(ast.ArrayTerm(path, input)); err != nil {
```

Comment:
Instead of passing `path` as an argument, could we simplify the function signature of `walkNoPath`, and just pass a constant empty array term here? Since we know the path member won't be used, maybe we can even get away with passing in `ast.Null` here.

Reply:
Alright. I've updated to pass an empty array term from a package var.